app/assets/javascripts/simple-data-grapher.js

```diff
@@ -0,0 +1,76 @@
+function determineType2(graphType){
+    if (graphType==="Horizontal" || graphType==="Vertical"){
```

Comment:
Maybe this would be better if it was a `switch` statement?

Reply:
That might address https://codeclimate.com/github/publiclab/plots2/pull/6113 asking about complexity here...